src/index.ts

```diff
@@ -40,7 +41,12 @@ const std = {
   isSubset(a: FiltrexType, b: FiltrexType) {
     const A = std.coerceArray(a);
     const B = std.coerceArray(b);
-    return +A.every((val) => B.includes(val));
+    return +A.every((val) => {
+      if (typeof val === 'object') {
+        return find(B, val) !== undefined;
+      }
```

Comment:
`lodash.find` doesn’t work with primitive data types,  so I have to do this check here.